python/cuml/test/test_multiclass.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import numpy as np
+import pytest
+from cuml import multiclass as cu_multiclass
+from cuml import LogisticRegression as cuLog
+from cuml.test.utils import unit_param
+from . test_linear_model import make_classification_dataset
+
+
+@pytest.mark.parametrize("strategy", ['ovr', 'ovo'])
+@pytest.mark.parametrize("use_wrapper", [True, False])
+@pytest.mark.parametrize("nrows", [unit_param(1000)])
+@pytest.mark.parametrize("num_classes", [3])
+@pytest.mark.parametrize("column_info", [unit_param([10, 4])])
```

Comment:
```suggestion
@pytest.mark.parametrize("nrows", [1000])
@pytest.mark.parametrize("num_classes", [3])
@pytest.mark.parametrize("column_info", [[10, 4]])
```
I dont think its necessary to mark these as `unit` if there are no `quality` or `stress` to go with it.